src/main/resources/apps/pantheon/modules/all/html.html

```diff
@@ -43,7 +43,7 @@ <h1>All Modules</h1>
                     ${item.pant:transientSourceName}
                 </td>
                 <td>
-                    <sly data-sly-use.date="${'com.redhat.pantheon.use.DateFormatter' @ format='yyyy/MM/dd HH:mm:ss', date=item.jcr:created}">${date.value}</sly>
+                    <!-- <sly data-sly-use.date="${'com.redhat.pantheon.use.DateFormatter' @ format='yyyy/MM/dd HH:mm:ss', date=item.jcr:created}">${date.value}</sly> -->
```

Comment:
Why is this commented? If this was just a mistake then can we bring it back?

Reply:
for some reason, modules.html throws 500 error on com.redhat.pantheon.use.DateFormatter. This branch is based off master. I commented it out so that I can see the modules.html page for the delete button. Let me uncomment it.

Reply:
done

Reply:
Hmm, well that's worrisome, but we can troubleshoot it later if necessary. Thanks for explaining!